RecipesLyricsMashup/RecipesLyricsMashupV2/RecipeLyricsMashup/RecipeLyricsMashup/RecipeLyricsMashup.csproj

```diff
@@ -4,6 +4,10 @@
     <TargetFramework>netcoreapp3.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="APIToken.txt" />

```

Comment:
I removed the API key from the project. I realize that this was already added to the GitIgnore. However, now that we have done our code reviews, aside from the professor, you may not want the API key to be posted in plain text on a document in a public repo.